core/templates/dev/head/profile/Signup.js

```diff
@@ -75,7 +75,7 @@ oppia.controller('Signup', [
           username: $scope.username
         }).then(function(response) {
           if (response.data.username_is_taken) {
-            $scope.warningText = 'Sorry, this username is already taken.';
+            $scope.warningText = 'I18N_SIGNUP_ERROR_USERNAME_TAKEN';
```

Comment:
big error here -- need to change all instances of warningText to warningI18nCode